badges/oss-library-night-24/oss-library-night-24.ts

```diff
@@ -49,7 +49,7 @@ export default define({
     if (pulls.length > 0) {
       grant(
         'oss-library-night-24',
-        "I've participated in the [Opensource Library Night 24](https://events.yandex.ru/events/opensourcenight)!",
+        'I\'ve participated in the <a href="https://events.yandex.ru/events/opensourcenight">Opensource Library Night 24</a>!',
```

Comment:
@antongolub Description should NOT contain any HTML entries. As those are added to the `title=""` attribute of the badge.

Reply:
<img width="347" alt="image" src="https://github.com/user-attachments/assets/e6353dec-a1f5-4102-8c1b-92037bf54357" />

Reply:
It's rendered inside `<strong>` tag, so this produces the crappy output. Affects `my-badges/my-badges-contributor` too. We'll fix.